public/app/features/admin/ldap/LdapGroupMapping.tsx

```diff
@@ -0,0 +1,71 @@
+import { useFormContext } from 'react-hook-form';
+
+import { SelectableValue } from '@grafana/data';
+import { Box, Button, Field, Input, RadioButtonGroup, Switch } from '@grafana/ui';
+import { t, Trans } from 'app/core/internationalization';
+import { LdapPayload, OrgRole } from 'app/types';
+
+const roleOptions: Array<SelectableValue<string>> = Object.keys(OrgRole).map((key) => {
+  return { label: key, value: key };
+});
+
+interface GroupMappingProps {
+  onRemove: () => void;
+  groupMappingIndex: number;
+}
+
+export const GroupMappingComponent = ({ groupMappingIndex, onRemove }: GroupMappingProps) => {
+  const { getValues, register, setValue } = useFormContext<LdapPayload>();
+  return (
+    <Box borderColor="strong" borderStyle="solid" padding={2} marginBottom={2}>
+      <Field
+        htmlFor="group-dn"
+        label={t('ldap-drawer.group-mapping.group-dn.label', 'Group DN')}
+        description={t(
+          'ldap-drawer.group-mapping.group-dn.description',
+          'The name of the key used to extract the ID token from the returned OAuth2 token.'
+        )}
+      >
+        <Input id="group-dn" {...register(`settings.config.servers.0.group_mappings.${groupMappingIndex}.group_dn`)} />
+      </Field>
+      <Field label={t('ldap-drawer.group-mapping.org-role.label', 'Org role *')}>
+        <RadioButtonGroup
+          id={`org-role-${groupMappingIndex}`}
+          options={roleOptions}
+          value={getValues(`settings.config.servers.0.group_mappings.${groupMappingIndex}.org_role`)}
+          onChange={(v) => setValue(`settings.config.servers.0.group_mappings.${groupMappingIndex}.org_role`, v)}
+        />
+      </Field>
+      <Field
+        htmlFor="org-id"
+        label={t('ldap-drawer.group-mapping.org-id.label', 'Org ID')}
+        description={t(
+          'ldap-drawer.group-mapping.org-id.description',
+          'The Grafana organization database id. Default org (ID 1) will be used if left out'
+        )}
+      >
+        <Input
+          id="org-id"
+          type="number"
+          {...register(`settings.config.servers.0.group_mappings.${groupMappingIndex}.org_id`, { valueAsNumber: true })}
+        />
+      </Field>
+      <Field
```

Comment:
Could you please only show this if the current user is a Grafana Admin? (To not show it on Cloud)

Reply:
It was changed as requested. PTAL?